lib/compress/zstd_cwksp.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2016-present, Yann Collet, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under both the BSD-style license (found in the
+ * LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ * in the COPYING file in the root directory of this source tree).
+ * You may select, at your option, one of the above-listed licenses.
+ */
+
+#include "zstd_cwksp.h"
+
+#if defined (__cplusplus)
+extern "C" {
+#endif
+
+/* all functions have been inlined... */
```

Comment:
very minor: 
some compiler(s) may complain about an empty unit.
I remember having to declare a useless symbol once in a similar case to avoid that.